Automattic/woocommerce-payments
# fix: onboarding currency messaging for BNPLs

Fixes #

#### Changes proposed in this Pull Request









I made some changes to the messaging, to ensure that we're not adding a bunch of unused currencies when a BNPL method is enabled.

I haven't fixed the backend logic, yet. This is just a fix on the messaging, specifically on the notice displayed on the onboarding page.

This is what was displayed before these changes:
![Screenshot 2024-02-14 at 6 47 07 PM](https://github.com/Automattic/woocommerce-payments/assets/273592/ed2dd578-1e33-484a-9668-abea583d0c1a)
![Screenshot 2024-02-14 at 6 47 01 PM](https://github.com/Automattic/woocommerce-payments/assets/273592/2f9635b2-e803-49b1-82d8-58ed75f824d8)
![Screenshot 2024-02-14 at 6 46 55 PM](https://github.com/Automattic/woocommerce-payments/assets/273592/2291dfc9-f7d3-4cfd-896d-0fe111ad95f1)

#### Testing instructions





- Ensure you have Klarna, Afterpay/Clearpay, Affirm disabled
- Take note of your Stripe account currency. Let's say it's USD.
- Make sure you have only USD (as a default currency) in the multi-currency settings
- Go to http://wcpay.test/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fadditional-payment-methods
- Enable any of the BNPL methods
- There should be no messaging displayed
- Now change the store currency in http://wcpay.test/wp-admin/admin.php?page=wc-settings&tab=general to another currency - let's say EUR
- Go to http://wcpay.test/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fadditional-payment-methods
- Enable any of the BNPL methods
- There should be messaging saying that USD will be added as a currency

![Screenshot 2024-02-14 at 6 34 54 PM](https://github.com/Automattic/woocommerce-payments/assets/273592/baf8be7a-283e-4559-a00f-c408e61f605f)
![Screenshot 2024-02-14 at 6 34 48 PM](https://github.com/Automattic/woocommerce-payments/assets/273592/fcfeec19-b3c0-444a-8d37-a8bc77859956)
![Screenshot 2024-02-14 at 6 34 41 PM](https://github.com/Automattic/woocommerce-payments/assets/273592/cfaf5571-0e40-4ede-be87-ec614fe252e6)


-------------------


**Post merge**
